api/config/config_staging.json

```diff
@@ -69,7 +69,7 @@
   },
   "server": {
     "apiBaseUrl": "https:\/\/api-dot-all-of-us-rw-staging.appspot.com",
```

Comment:
does apiBaseUrl also need to change?

Reply:
Please also confirm these values:
* `:api_endpoint_host` in `environments.rb`
* `allOfUsApiUrl` in `ui/src/environments/environment.staging.ts`
* the `Content-Security-Policy-Report-Only` list in `ui/app.yaml`

Reply:
@jmthibault79 API is not changed for now. They may do it later.

Reply:
The API change is #7056